app/bundles/CampaignBundle/Executioner/KickoffExecutioner.php

```diff
@@ -129,8 +129,6 @@ private function prepareForExecution()
     {
         $this->logger->debug('CAMPAIGN: Triggering kickoff events');
 
-        $this->progressBar  = null;
```

Comment:
I'm not sure about this. It's suppose to reset the progress bar. How is it related to this PR?

Reply:
@escopecz the property is immediately initialized just a little bit below.